quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/dist/approvals/cli/help/HelpCommandDistTest.testExportHelpAll.approved.txt

```diff
@@ -208,37 +208,35 @@ Tracing (Preview):
 --tracing-compression <method>
                      Preview: OpenTelemetry compression method used to compress payloads. If unset,
                        compression is disabled. Possible values are: gzip, none. Default: none.
-                       Available only when Tracing is enabled.
---tracing-enabled <true|false>
-                     Preview: Enables the OpenTelemetry tracing. Default: false.
```

Comment:
I see you removed the `tracing-enabled` option in exchange for the feature. 

I assume we want to keep the feature flag only as long as it is in preview. Once the preview is removed we would need the `tracing-enabled` again. 

So I'd suggest to keep it. This would be similar to metrics enabled, as we don't have a metrics feature to be enabled for it.